Modules/Module_Processing/Module_ASL/xASL_wrp_ResampleASL.m

```diff
@@ -246,19 +246,24 @@ function xASL_wrp_ResampleASL(x)
 
     % Save PWI4D (subtracted only)
     fprintf('%s\n', PathPWI4D{iSpace});
-    xASL_io_SaveNifti(PathASL4D{iSpace}, PathPWI4D{iSpace}, PWI4D, 32, false);
-    spm_jsonwrite(PathPWI4Djson{iSpace}, x.Q);
+	jsonFields.EchoTime = x.Q.EchoTime_PWI4D;

```

Comment:
This is incorrect, we should save PWI4D as PWI4D, PWI3D as PWI3D, PWI as PWI A single JSON may have all three. Now you are renaming them, that doesn't work. Plus, there is no BIDS standard for derivatives yet, so moving to BIDS rawdata standard doesn't make sense for derivatives necessarily.